homeworks/lesson2/app/src/components/login/login.jsx

```diff
@@ -0,0 +1,42 @@
+import React, {Component} from 'react';
+import {Button, FormGroup, FormControl, ControlLabel} from 'react-bootstrap';
+import './login.css';
+
+class LoginForm extends Component {
+
+    handleSubmit = event => {
```

Comment:
Please declare methods which you are using like extra below than render, lifecycles methods, constructor, static methods of a component.

Reply:
Should I just declare empty lifecycle methods?

Reply:
Nope. That is needed just for other developers who can use that component. If them developers will see at once all interfaces of that component (static methods, propTypes and other) than developers can use it and will do not spend the time to search interface of that.